momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -473,10 +473,26 @@ public CompletableFuture<CacheIncrementResponse> increment(
    * @return Future containing the result of the add element operation.
    */
   public CompletableFuture<CacheSetAddElementResponse> setAddElement(
-      String cacheName, String setName, String element, CollectionTtl ttl) {
+      String cacheName, String setName, String element, @Nullable CollectionTtl ttl) {
     return scsDataClient.setAddElement(cacheName, setName, element, ttl);
   }
 
+  /**
+   * Add an element to a set in the cache.
+   *
+   * <p>After this operation the set will contain the union of the element passed in and the
+   * original elements of the set.
+   *
+   * @param cacheName Name of the cache to store the item in
+   * @param setName The set to add the element to.
+   * @param element The data to add to the set.
+   * @return Future containing the result of the add element operation.
+   */
+  public CompletableFuture<CacheSetAddElementResponse> setAddElement(
+      String cacheName, String setName, String element) {
+    return scsDataClient.setAddElement(cacheName, setName, element, null);
```

Comment:
I called the functions inside the cache client for these instead of passing the null ttls to sdsDataClient. Will approve this change and make a PR to be consistent with yours, since I think this way is better.